integration

```diff
@@ -479,6 +479,7 @@
   "jihao/rokid-webhook-hass",
   "jippi/hass-nordnet",
   "jjlawren/sonos_cloud",
+  "jlinford/ha-gpio",
```

Comment:
Is this something you are still working on?
The 2 issues that were opened since this PR was closed by your bot pending your response.